assets/scss/layout/_reset-password.scss

```diff
@@ -0,0 +1,48 @@
+.page--reset-password,
```

Comment:
A question for the other reviewers, existing css in bolt seems inspired by http://getbem.com/naming/ or a similar scheme, I haven't seen it documented explicitly anywhere. Is it a requirement? (I think that might make sense, but I'm not certain I've followed it myself...)

Reply:
i basically copied that from https://github.com/bolt/core/blob/master/assets/scss/layout/_login.scss
But I can confirm that the whole SCSS naming scheme can be improved. Would rather handle that in a separate PR though